ingestion-beam/src/test/java/com/mozilla/telemetry/DecoderMainTest.java

```diff
@@ -74,28 +74,6 @@ public void testMixedErrorCases() throws Exception {
         matchesInAnyOrder(expectedErrorOutputLines));
   }
 
-  @Test
```

Comment:
I've removed gzip-specific cases out of the integration tests in favor of having the main `input.ndjson` contain one gzipped document.